html_css/intermediate_html/form-validations.md

```diff
@@ -186,9 +186,9 @@ It's also worth noting client-side validations are not a silver bullet for ensur
 <div class="lesson-content__panel" markdown="1">
 
 1. Read and follow along to [MDN's Client-Side Form Validation Guide](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation)
-2. SitePoint's [Complete Guide to HTML Forms and Constraint Validation Guide](https://www.sitepoint.com/html-forms-constraint-validation-complete-guide/)
-3. Watch [Web Dev Simplified's JavaScript Form Validations Tutorial](https://www.youtube.com/watch?v=Dorf8i6lCuk)
-4. Read CSS Tricks brilliant [Form Validation UX in HTML and CSS Guide](https://css-tricks.com/form-validation-ux-html-css/)
+    * Note: Halfway through, the doc covers the Constraint Validation API: a way to validate forms with javascript. We'll cover that in a future lesson, you can skip that part for now.
```

Comment:
This doesn't need the "Note" text:

```suggestion
    * Skip the section on "Validating forms using JavaScript". This will be covered in a future lesson.
```